package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "binaryen.ml",
+  "name": "@grain/binaryen",
```

Comment:
I wonder if we should call it `@grain/binaryen-ml` since it's going on npm (and not to be confusing that it's not JS).

Reply:
I've changed it, but I don't think it matters because:
1. Scoped modules don't show up in search by default
2. If you try to install this with npm, it will crash and fail
3. The description and keywords both say "OCaml"

Reply:
Either way!